scipy/linalg/basic.py

```diff
@@ -70,11 +70,14 @@ def solve(a, b, sym_pos=False, lower=False, overwrite_a=False,
         Input data for the right hand side.
     sym_pos : bool, optional
         Assume `a` is symmetric and positive definite. This key is deprecated
-        and assume_a = 'pos' keyword is recommended instead. The functionality
-        is the same. It will be removed in the future.
+        and ``assume_a='pos'`` is recommended instead. The functionality is
+        the same. It will be removed in the future.
+        Default is False.
     lower : bool, optional
-        If True, only the data contained in the lower triangle of `a`. Default
-        is to use upper triangle. (ignored for ``'gen'``)
+        If True, the calulation is based only on the data in the lower triangle of
```

Comment:
This line is a few characters too long; it's causing the failure in "scipy.scipy (Main Lint)"
![image](https://user-images.githubusercontent.com/6570539/125844496-0f0a92cd-a974-45f7-bb93-35742d87398f.png)

Reply:
True, I didn't see that. I'll fix it